code/mission/missionparse.cpp

```diff
@@ -4602,7 +4602,7 @@ void parse_wing(mission *pm)
 	}
 
 	if (optional_string("+Flags:")) {
-		auto count = (int)stuff_string_list( wing_flag_strings, PARSEABLE_WING_FLAGS);
+		auto count = (int) stuff_string_list(wing_flag_strings, PARSEABLE_WING_FLAGS);
```

Comment:
I'm iffy on this whitespace change being mixed in with the more impactful change, but it's decent style cleanup. Up to you if you want to keep it or not, just something I wanted to prompt a second look at.